tests/testthat/test-target_market_share.R

```diff
@@ -1175,3 +1177,135 @@ test_that("`target_market_share` outputs as expected for companies with 0
   expect_equal(out$projected$technology_share, c(0, 1))
   expect_equal(out$target_sds$technology_share, c(0, 0))
 })
+
+test_that("outputs columns `percent_change_by_scope` and `scope`", {
+  out <- target_market_share(
+    fake_matched(),
+    fake_ald(),
+    fake_scenario(),
+    region_isos_stable
+  )
+
+  expected_added_columns <- c( "scope", "percentage_of_initial_production_by_scope")
+
+  expect_equal(setdiff(expected_added_columns, names(out)),character(0))
+})
+
+test_that("with known input outputs `percent_of_initial_production_by_scope` as
+          expected, at portfolio level", {
```

Comment:
something's weird with the indentation here?

Reply:
it's more below.... looks like the entire function is indented multiple tabs?

Reply:
hmmm weird it looks ok in my RStudio?